core/src/Baked/Theme/Default/DomainComponents.cs

```diff
@@ -140,6 +140,12 @@ public static DataTable.Footer MethodDataTableFooter(MethodModel method, Compone
         return B.DataTableFooter(l($"{method.Name}.FooterLabel"), options: options);
     }
 
+    public static ComponentDescriptor<Button> MethodButton(MethodModel method, ComponentContext context,
+        Action<Button>? options = default
+    ) => B.Button(method.Name, method.GetRequiredSchema<RemoteAction>(context),
```

Comment:
context should have been drilled at the top of this builder method.

`context = context.Drill(nameof(Button))`

plus, should have been drilled when getting schema for action `context.Drill(nameof(Button.Action))`

so that it becomes `.../button/action`